python/postprocessing/modules/jme/jetmetUncertainties.py

```diff
@@ -64,7 +68,7 @@ def __init__(self, era, globalTag, jesUncertainties = [ "Total" ], jetType = "AK
         #W-tagging PUPPI softdrop JMS values: https://twiki.cern.ch/twiki/bin/view/CMS/JetWtagging
         #2016 values - use for 2018 until new values available
         self.jmsVals = [1.00, 0.9906, 1.0094] #nominal, down, up
-        if self.era == "2017":
+        if self.era in ["2017","2018"]:
```

Comment:
This is a good question for @clelange.

Reply:
Using 2017 for 2018 was the recommendation given to me for the Full Run II b* analysis but I don't know if we can call it official. I think using 2017 for 2018 is better motivated than using 2016 or nothing. @clelange did you have an opportunity to think about this?

Reply:
@clelange can you comment?

Reply:
After discussing this with the other JMAR convener, we've agreed that we should indeed use the 2017 JMR and JMS values for 2018 (for now), since the pileup profile is more similar. So @lcorcodilos, could you edit lines 33-37 to reflect this?

Reply:
It looks like @fgolf just merged the PR without these changes, so I will make them tomorrow.

Reply:
@fgolf, I just made a PR (#180 )to address this.